app/controllers/albums.js

```diff
@@ -0,0 +1,16 @@
+const { getAlbums, getPhotos } = require('../services/albums');
+
+exports.listAlbums = (req, res, next) => {
+  getAlbums()
```

Comment:
we always have to return promises. if we dont do that Node dont wait promises to finish and continue the execution of next lines